Header header  # timestamp
geometry_msgs/Vector3 x1  # Estimated x1
geometry_msgs/Vector3 x2_prime  # Estimated x2_prime
geometry_msgs/Vector3 x2  # Estimated x2